ts/index.ts

```diff
@@ -153,9 +199,55 @@ export class DexieStorageBackend extends backend.StorageBackend {
         // return deletedCount
     }
 
-    async count(collection : string, query) {
+    async countObjects(collection : string, query) {
         return this.dexie.collection(collection).count(query)
     }
+
+    async suggestObjects<S, P = any>(collection : string, query, options: backend.SuggestOptions = {}) {
```

Comment:
As noted in your Storex core PR, this would be nice to have in a higher-level search package. It does seem some lower-level things are used, so we'd have to think about how to expose these as primitives, or pack them as find* options.